core/pyproject.toml

```diff
@@ -42,6 +42,7 @@ PyMySQL = { version="^1.0.2", optional = true }
 redis = { version = "^4.3.5", optional = true }
 SQLAlchemy = { version = "^2.0.4", optional = true }
 sqlglot = "^10.4.3"
+fastparquet = "^2023.2.0"
```

Comment:
let's keep it alphabetically sorted.

Reply:
is it an optional extra? we should figure out if we want to name the extras after the block name. would make it easier to optionally install based on used blocks in the future.